### PR TITLE
Update community operator 0.4.0 CSV fields

### PR DESCRIPTION
**Pull request by miguelsorianod**

### CODE REVIEW

**Comment by miguelsorianod** [comment]
Ready for review @eguzki . I've applied those changes and some more that I've detected doing a diff.
There's also the decision on whether we should set maturity to alpha (in prod is set like that) or stable (right now in this branch)

File: deploy/olm-catalog/3scale-operator/3scale-community-operator.v0.4.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -164,6 +163,7 @@ spec:
                 - name: ZYNC_IMAGE
                   value: "quay.io/3scale/3scale27:zync-3scale-2.7.0-GA"
                 image: quay.io/3scale/3scale-operator:v0.4.0
+                imagePullPolicy: Always
```
Comment (eguzki) [inline deploy/olm-catalog/3scale-operator/3scale-community-operator.v0.4.0.clusterserviceversion.yaml]: why? 

This tag is not floating tag and default pull policy `IfNotPresent` is good enough

Comment (miguelsorianod) [reply deploy/olm-catalog/3scale-operator/3scale-community-operator.v0.4.0.clusterserviceversion.yaml]: This was to match what's in the 2.7-stable-prod branch

Comment (eguzki) [reply deploy/olm-catalog/3scale-operator/3scale-community-operator.v0.4.0.clusterserviceversion.yaml]: If the tag is not floating, I would go for default pull policy

Comment (miguelsorianod) [reply deploy/olm-catalog/3scale-operator/3scale-community-operator.v0.4.0.clusterserviceversion.yaml]: :+1: done

**Comment by miguelsorianod** [comment]
>  There's also the decision on whether we should set maturity to alpha (in prod is set like that) or stable (right now in this branch).

I set it to stable and I'll update the master branch to have it in stable too.
I'll also remove imagePullPolicy:always from master branch

**Pull request merged**